### DISCUSSION TITLE
Multi-Display Support

### DISCUSSION THREAD

**Discussion by asundust** [mulaRahul/keyviz#21] [Ideas]
因为主屏幕可能影响到当前的工作

**Comment by mulaRahul** [comment]
Indeed, there should be an option to put the visualization on desired display, if there are _multiple_ displays. I'll make sure to list this in **future plans**! Flutter (the framework used to build the program) doesn't support explicit positioning of the application window on multiple displays, that's why it's not added from the get-go.

**Comment by mulaRahul** [comment]
Added with [v2.0.0a2](https://github.com/mulaRahul/keyviz/releases/tag/v2.0.0a2)
